philander/systypes.py

```diff
@@ -32,6 +32,9 @@ class ErrorCode(Enum):
     errFailure			= 20	# Action failed, but might be successful in next trials.
     errMalfunction		= 21	# (Persistent) defect of underlying HW.
 
+    def is_ok(self):
```

Comment:
Rethink the naming. To follow the convention, it should be `isOk()`.
Implementation is suspect. Seems to be an infinite recursion. Make sure, a boolean is returned.
Misses documentation.